library/styleguideComponents/ReactComponent/index.js

```diff
@@ -27,6 +29,13 @@ export default class ReactComponent extends Component {
         this.setState({ activeTabName: tab });
     }
 
+    editLinkClassName() {
+        const isExpanded = window.location.href.includes('/#!/');
+        return classnames('react-rainbow-component_example-edit-link-container', {
+            'react-rainbow-component_example-edit-link-container--compress': isExpanded,
+        });
+    }
```

Comment:
You can put this function outside the component since don't use this. Also this enables you to remove the eslint rule added.